changelog/upgrade-notes.md

```diff
@@ -2,6 +2,26 @@
 
 ### Next release
 
+#### Simpler `constrainResolution` configuration
+
+The `constrainResolution` configuration for `ol.interaction.PinchZoom` and `ol.interaction.MouseWheelZoom`
+can now be set directly with an option in `ol.interaction.defaults`:
+```js
+ol.interaction.defaults({
+  constrainResolution: true
+});
+```
+
+#### `ol.interaction.MouseWheelZoom` no longer zooms to a whole-number zoom level
```

Comment:
This change was introduced with v3.20.0 already, so it should not need a mention here.